### PR TITLE
CBV-64 Added styling to landing page

### PR DESCRIPTION
**Pull request by christiancordova107**
image

### CODE REVIEW

File: src/entrypoint/index.tsx [hunk 1]
```diff
@@ -28,7 +29,13 @@ function NewProject() {
     }
 
     return (
-        <div className='task'>
+        <div className='task' style={{
```
Comment (matchagreen) [inline src/entrypoint/index.tsx]: You should put the styling on the `index.css` file as opposed to the .tsx file

File: src/entrypoint/index.tsx [hunk 2]
```diff
@@ -28,7 +29,13 @@ function NewProject() {
...
+            backgroundImage: `url(${tankBluePrint})`,
```
Comment (matchagreen) [inline src/entrypoint/index.tsx]: The while lines in the tank distract too much from the foreground. Try dimming the image down.

**Review by matchagreen — CHANGES_REQUESTED** [review]

**Comment by christiancordova107** [comment]
image
new background image

File: src/entrypoint/index.tsx [hunk 3]
```diff
@@ -54,11 +54,11 @@ function NewProject() {
                 <i style={{ paddingRight: "10px" }}
```
Comment (matchagreen) [inline src/entrypoint/index.tsx]: You should be moving the style to the css file

**Review by matchagreen — CHANGES_REQUESTED** [review]
Are we using the tank image for anything else? if not we should remove it before merging

**Review by matchagreen — APPROVED** [review]

**Review by garciagitmauricio — APPROVED** [review]

**Pull request merged**